packages/edit-site/src/components/dataviews/provider.js

```diff
@@ -25,7 +25,7 @@ const { useLocation } = unlock( routerPrivateApis );
 // (entity request and useEffect to update the view).
```

Comment:
Why we have this provider component? What's its purpose?

Reply:
Share the state between the sidebar and the main area.